nixos/modules/services/misc/spice-webdavd.nix

```
@@ -0,0 +1,34 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+let
+  cfg = config.services.spice-webdavd;
+in
+{
+  options = {
+    services.spice-webdavd = {
+      enable = mkEnableOption "Spice guest webdav proxy daemon";
```

[src](https://github.com/nixos/nixpkgs/blob/d8e52b5b53cf58babb1f48c9e7e28ab0a198a219/lib/options.nix#L98)
```suggestion
      enable = mkEnableOption "the spice guest webdav proxy daemon";
```